KSTokenView/KSTokenView.swift

```diff
@@ -419,7 +426,7 @@ open class KSTokenView: UIView {
    }
    
     override open var intrinsicContentSize : CGSize {
-      return CGSize(width: UIViewNoIntrinsicMetric, height: _intrinsicContentHeight)
+        return CGSize(width: UIViewNoIntrinsicMetric, height: _intrinsicContentHeight)
```

Comment:
this already seems indented with 2 spaces, please verify
or are we keeping 4space/tab indentation for KSTokenView as changing to 2 spaces will show a lot of diff

Reply:
For pods that are not ours, we should respect their indentation.
For KSTokenView, it is 3 spaces.

So making it 3 spaces.